app/armor/ammunition.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+
+class Armour:
+    def __init__(self, part: str, protection: int) -> None:
+        self.part = part
+        self.protection = protection
+
+    @classmethod
+    def armours_from_dict(
+        cls,
+        knight_name: str,
+        knightsConfig: list[dict]
```

Comment:
Accordingly, to flake8 rules, all variables have to be named in lowercase format, fix that.